webpack/src/Router/HostReports/IndexPage/Components/HostReportsTable/HostReportsTable.js

```diff
@@ -55,11 +54,11 @@ const HostReportsTable = ({
       <HostReportDeleteModal toDelete={toDelete} onSuccess={onDeleteSuccess} />
       {body}
       <Pagination
-        viewType="list"
         itemCount={itemCount}
-        pagination={pagination}
-        onChange={fetchAndPush}
-        dropdownButtonId="host-reports-page-pagination-dropdown"
+        /* eslint-disable-next-line camelcase */
+        onChange={({ page, per_page }) =>
+          fetchAndPush({ page, perPage: per_page })
```

Comment:
I believe this should be `onChange={fetchAndPush}`
and in `fetchAndPush` we should use only `per_page`
as in https://github.com/theforeman/foreman_host_reports/blob/develop/webpack/src/Router/HostReports/IndexPage/IndexPageActions.js#L36 we are sending `per_page` to the server.

Reply:
Alright then, I'll update that here and in webhooks as well :)